docker-compose.yml

```
@@ -190,6 +190,9 @@ services:
     healthcheck:
       <<: *healthcheck_defaults
       test: ["CMD-SHELL", "nc -z localhost 9092"]
+      interval: 3s
+      timeout: 600s
+      retries: 200
```

Is this the best way to structure this? From the [docs](https://docs.docker.com/engine/reference/builder/#healthcheck): 

> The health check will first run **interval** seconds after the container is started, and then again **interval** seconds after each previous check completes.
>
> If a single run of the check takes longer than **timeout** seconds then the check is considered to have failed.
>
> It takes **retries** consecutive failures of the health check for the container to be considered `unhealthy`.

The defaults are:

```
--interval=DURATION (default: 30s)
--timeout=DURATION (default: 30s)
--retries=N (default: 3)
```

This means currently the check is running every 30 seconds, each check is timing out after 30 seconds, and we're checking three times before giving up and getting an `unhealthy`. With this PR we would run the check every three seconds, each check out would timeout after 5 minutes, and we would try 200 times before giving up. This feels weird, 603 s × 200 = 120,600 s / 60 / 60 = 33.5 h. Is that what's intended here?

---

Good point!

I copied the settings from our [existing ones fro clickhouse](https://github.com/getsentry/self-hosted/blob/fcf0d3599b839535f19aa9bc96bc4b7af942a992/docker-compose.yml#L226), so I guess both should be updated.

---

Ah, good call. Yeah let's update both. I think the intention is more like:

```suggestion
      interval: 3s
      timeout: 3s
      retries: 100
```

(3 s + 3 s) * 100 = 600 s overall. What do you think?

---

10 / 10 / 30 also works. :o)

---

I like 10/10/30 a bit better!